sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1082,6 +1082,14 @@ object functions {
    */
   def isnan(e: Column): Column = withExpr { IsNaN(e.expr) }
 
+  /**
+   * Return true iff the column is NaN.
+   *
+   * @group normal_funcs
+   * @since 1.6.0
+   */
+  def isnan(columnName: String): Column = isnan(Column(columnName))
```

Comment:
We won't add this per the comments on the top of this file.

Reply:
https://github.com/apache/spark/blob/f8b1424d2f51bc8a5b500c70742be8a9dfffa1df/sql/core/src/main/scala/org/apache/spark/sql/functions.scala#L58-L60